src/lib/interceptor/search-request.interceptor.spec.ts

```diff
@@ -66,8 +66,6 @@ describe('SearchRequestInterceptor', () => {
         it('return search request dto when input is valid', async () => {
             expect(await interceptor.validateBody({ select: ['col1', 'col2'] })).toEqual({
                 select: ['col1', 'col2'],
-                take: 20,
-                withDeleted: false,
```

Comment:
During the validation stage, take and withDeleted were being calculated with default values, and this was being serialized in the query(nextCursor). Now, the calculations for search option values are done only in the `intercept` method.